src/app/content/highlights/reducer.spec.ts

```diff
@@ -406,6 +407,36 @@ describe('highlight reducer', () => {
       expect(state.summary.highlights).toMatchObject(highlights);
       expect(state.summary.loading).toEqual(false);
     });
+
+    it('noops for receive summary highlights with stale filters', () => {
+      const highlights: SummaryHighlights = {
+        chapter_id: {
+          page_id: [
+            {id: 'highlight'} as HighlightData,
+          ],
+        },
+      };
+
+      const mockState = {
+        ...initialState,
+        summary: {
+          ...initialState.summary,
+          filters: {
+            colors: [HighlightColorEnum.Green],
+            locationIds: ['id'],
+          },
+          loading: true,
+        },
+      };
+
+      const staleFilters = {} as any as SummaryFilters;
```

Comment:
You could add `locationIds: [], colors: []` into the object and it wouldn't need to be casted.